lib/utils/debounce.js

```diff
@@ -112,3 +113,42 @@ export class Debouncer {
     return debouncer;
   }
 }
+
+let debouncerQueue = new Set();
+
+/**
+ * Adds a `Debouncer` to a list of globally flushable tasks.
+ *
+ * @param {!Debouncer} debouncer Debouncer to enqueue
+ * @return {void}
+ */
+export const enqueueDebouncer = function(debouncer) {
+  // Re-enqueued debouncers are put at the end of the queue; for Set, this
+  // means removing and re-adding, since forEach traverses insertion order
+  if (debouncerQueue.has(debouncer)) {
```

Comment:
The expected behavior is FIFO so to match the way the array was processed previously, this shouldn't be deleted.

Reply:
Updated